docs/sources/dirtyduck/triage/experiments/inspections_dt.yaml

```diff
@@ -184,6 +185,9 @@ feature_group_definition:
 
 feature_group_strategies: ['all']
 
+prediction:
+   rank_tiebreaker: "best"
```

Comment:
maybe add a comment warning that this is illustrative, but you might not want to use `best` in most cases unless you have good reason to do so...

Reply:
I actually left this in accidentally from some speed testing I was doing. I'll remove it and let them use the default (unless @nanounanue  wants to change it to something else)